sdk/storage/azure-storage-blob/azure/storage/blob/blob_client.py

```diff
@@ -1412,17 +1421,25 @@ def _abort_copy_options(self, copy_id, **kwargs):
 
     @distributed_trace
     def abort_copy(self, copy_id, **kwargs):
-        # type: (Union[str, FileProperties], **Any) -> None
+        # type: (Union[str, BlobProperties], **Any) -> None
         """Abort an ongoing copy operation.
 
         This will leave a destination blob with zero length and full metadata.
         This will raise an error if the copy operation has already ended.
 
         :param copy_id:
             The copy operation to abort. This can be either an ID, or an
-            instance of FileProperties.
-        :type copy_id: str or ~azure.storage.file.models.FileProperties
+            instance of BlobProperties.
+        :type copy_id: str or ~azure.storage.blob.models.BlobProperties
         :rtype: None
+
+        Example:
+            .. literalinclude:: ../tests/test_blob_samples_common.py
+                :start-after: [START abort_copy_blob_from_url]
+                :end-before: [END abort_copy_blob_from_url]
+                :language: python
+                :dedent: 12
```

Comment:
same here...should dedent be 8?

Reply:
12 looks good in the local docs here as well.